### DISCUSSION TITLE
DeepSpeed with StableDiffusion v2.1

### DISCUSSION THREAD

**Discussion by saqibameen** [deepspeedai/DeepSpeed#5166] [Q&A]
I am running SDv2.1 locally (https://github.com/Stability-AI/stablediffusion). Looking to integrate DeepSpeed in it. Not sure how can I go about doin that. Any pointers/directions would be appreciated.

**Comment by loadams** [comment]
Some pointers would be here, for our SD unit test coverage: https://github.com/microsoft/DeepSpeed/blob/master/.github/workflows/nv-sd.yml#L65

Not sure if you're looking to speed up inference or what the goal is, but the general getting started page would be a start: https://www.deepspeed.ai/getting-started/#writing-deepspeed-models
